config/cht/.eslintrc

```diff
@@ -0,0 +1,27 @@
+{
+  "env": {
+    "es6": true,
+    "node": true
+  },
+  "extends": "eslint:recommended",
+  "parserOptions": {
+    "ecmaVersion": 6,
+    "sourceType": "module"
+  },
+  "rules": {
+    "no-undef": "off",
```

Comment:
Should we turn these two back on as part of this review? `no-undef` and `no-unused-vars`?

Reply:
Done.